### ISSUE TITLE
FormSegmentedElement wrong in 6.3.3

### ISSUE THREAD

**Issue by alexodus** [TheJuki/KFormMaster#108]
strange behavior: in 6.3.3 when I click on item of FormSegmentedElement the component does not detect the change

```
valueObservers.add { newValue, element ->
                    ... //unreachable the first time we change item
                }
```

The 6.3.2 version is OK.

**Comment by TheJuki** [comment]
I just released [6.3.4](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if changing the value works for you.

**Comment by TheJuki** [comment]
I am closing this issue. If you have any issues feel free to reopen.
